install/changelog.md

```diff
@@ -52,6 +57,86 @@ title: "Change Log"
 
 > See [known issues](../reference/known-issues.html) for currently known issues and workarounds.
 
+## v0.15.0 {#v150}
+
+Released on August 13, 2018
+
+In addition to the 0.15.0 CLI release, we've released new versions of all of our packages.
+
+### Pulumi CLI
+
+This release brings many large improvements to the Pulumi CLI. Major features include:
+
+#### Parallelism
+
+Pulumi now preforms resource operations in parallel, when possible. By default, Pulumi allows ten parallel operations, but this can be controlled with the `-p` flag to `pulumi`. Presently, deletes are still processed sequentaly, but this will be changed in a future release. It is possible that your program may have implicit dependencies between two resources and the change to preform operations in parallel by default will cause issues. If this happens, we recommend that you set dependsOn on property in `resourceOptions` parameter to any resource. You may also pass `-p=1` to disable parallelism. Parallelism is only supported for nodejs programs today.
+
+#### First class providers
+
+Pulumi now allows resource providers to be created and configured as part of your program. In addition to the ambiently available provider for each resource, you can also create an explicit version of the provider and configure it as you wish. This is helpful, for example, if you want to create some resources in a different region from your main deployment. We have an [example](https://github.com/pulumi/pulumi-aws/blob/master/examples/multiple-regions/index.ts) that shows how to use this feature and we are excited to see what you build with it.
```

Comment:
Perhaps mention also “or configure the inputs to one provider (e.g. Kubernetes) using an output of a resource created by another (e.g.  Azure Kubernetes Service)”.